### PR TITLE
[zh-hant] Translation of libraries/bypass

### PR DESCRIPTION
**Pull request by lithiumpie**
- Translate libraries/bypass

### CODE REVIEW

File: zh-hant/lessons/libraries/bypass.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+---
+version: 1.0.0
```
Comment (doomspork) [inline zh-hant/lessons/libraries/bypass.md]: @vbrazo and @MikeRogers0 is there something wrong with our config file? These files should be excluded but they're not nor does it appear our `excluded_words` is working entirely:

https://github.com/elixirschool/elixirschool/blob/master/.typo-ci.yml#L37-L38

Comment (MikeRogers0) [reply zh-hant/lessons/libraries/bypass.md]: PR added https://github.com/elixirschool/elixirschool/pull/2135 :)

Comment (doomspork) [reply zh-hant/lessons/libraries/bypass.md]: Thank you @MikeRogers0 🎉

Comment (MikeRogers0) [reply zh-hant/lessons/libraries/bypass.md]: FYI if you merge in the latest master, it should update & not find any typos :)

**Comment by lithiumpie** [comment]
@doomspork please merge this PR if there are no more issues :P

**Pull request merged**